packages/slate-react/src/components/leaf.tsx

```diff
@@ -23,25 +23,6 @@ const Leaf = (props: {
     renderLeaf = (props: RenderLeafProps) => <DefaultLeaf {...props} />,
   } = props
 
-  const placeholderRef = useRef<HTMLSpanElement | null>(null)
-
-  useEffect(() => {
-    const placeholderEl = placeholderRef?.current
-    const editorEl = document.querySelector<HTMLDivElement>(
-      '[data-slate-editor="true"]'
-    )
-
-    if (!placeholderEl || !editorEl) {
-      return
-    }
-
-    editorEl.style.minHeight = `${placeholderEl.clientHeight}px`
-
-    return () => {
-      editorEl.style.minHeight = 'auto'
-    }
-  }, [placeholderRef])
-
```

Comment:
this can't be removed (it's needed to match the height of the editor to the height of the placeholder)